src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,42 @@
+@import "../../patternfly-utilities";
+
+$pf-l-gallery__item--Width: 250px !default;
+
+$pf-l-gallery--GridGap: 1rem !default;
+$pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax($pf-l-gallery__item--Width, 1fr)) !default;
+$pf-l-gallery__item--Flex: 0 1 $pf-l-gallery__item--Width !default;
```

Comment:
maybe we can avoid a shorthand here and declare a flex basis instead.

Reply:
Done.